disttask/framework/operator/data.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package operator
+
+// DataSource defines interface for source of all operators.
+// Operator read data from DataSource, then process the read data.
+type DataSource[T any] interface {
+	Start() error
+	Next() (T, error)
```

Comment:
How about renaming it to `Read()`? It should be complementary to `DataSink.Write()`.

Reply:
> How about renaming it to `Read()`? It should be complementary to `DataSink.Write()`.

Ok.